style.css

```diff
@@ -10,8 +10,13 @@
   text-decoration: none;
 }
 
```

Comment:
- [ ] When I first load the page it looks to be broken, because whole page a background layer: 
`* { background: #1b115e;}` When I delete it, the contents are visible. Please take a look at video I recorded and address this issue: 

https://user-images.githubusercontent.com/77038610/160784130-7930bfc1-752e-4dd0-aa86-e14ad5d16f6e.mp4


- [ ] You can give the background-color  to body element `body{ background-color: #1b115e}`